packages/payment-integration-api/src/CheckoutButtonProps.tsx

```diff
@@ -5,6 +5,7 @@ export default interface CheckoutButtonProps {
     containerId: string;
     checkoutService: CheckoutService;
     checkoutState: CheckoutSelectors;
+    className?: string;
```

Comment:
Can we give this prop a more descriptive name please.

Reply:
Done. Updated with `checkoutButtonContainerClass` naming